warehouse/templates/pages/help.html

```diff
@@ -269,6 +271,29 @@ <h3 id="utfdevices">{{ utfdevices() }}</h3>
         <p>Emerging solutions include biometric (facial and fingerprint) scanners and FIDO compatible credit cards. There is also growing support for <a href="https://fidoalliance.org/news-your-google-android-7-phone-is-now-a-fido2-security-key/" title="External link" target="_blank" rel="noopener">mobile phones to act as security devices</a>.</p>
         <p>As PyPI's two factor implementation follows the <a href="https://www.w3.org/TR/webauthn/" title="External link" target="_blank" rel="noopener">WebAuthn standard</a>, PyPI users will be able to take advantage of any future developments in this field.</p>
 
+        <h3 id="apitoken">{{ apitoken() }}</h3>
+
+        <p>API tokens provide an alternative way (instead of username and password) to authenticate when uploading packages to PyPI.</p>
+        <p>You can create a token for an entire PyPI account, in which case, the token will work for all projects associated with that account. Alternatively, you can limit a token's scope to a specific project.</p>
+        <p><strong>We strongly recommend you authenticate with an API token where possible.</strong></p>
+
+        <p>To make an API token:</p>
+
+        <ul>
+          <li><a href="#verified-email">Verify your email address</a> (check your <a href="{{ request.route_path('manage.account') }}">account settings</a>)</li>
+          <li>In your <a href="{{ request.route_path('manage.account') }}">account settings</a>), go to the API tokens section and select "Add API token"</li>
```

Comment:
Dangling paren. I made an amend to this commit https://github.com/pypa/warehouse/commit/d9b42b0450bf88e02d6d01ba581cfd7e08714306 to fix it, but I'm hesitant about force-pushing to your branch. Please feel free to rewrite history here or just add a new commit to fix the paren, your call. Sorry.

Reply:
No sweat, just pushed the change.